tests/cloud_health_test.py

```diff
@@ -0,0 +1,156 @@
+# -*- coding: utf-8 -*-
+#
+# © 2016 Krux Digital, Inc.
+#
+
+#
+# Standard libraries
+#
+
+from __future__ import absolute_import
+import unittest
+import sys
+
+#
+# Third party libraries
+#
+
+from mock import MagicMock, patch
+
+#
+# Internal libraries
+#
+
+from krux.cli import get_parser, get_group
+from krux.stats import DummyStatsClient
+from krux_cloud_health.cloud_health import CloudHealth, get_cloud_health, Interval, NAME
+
+
+class CloudHealthTest(unittest.TestCase):
+
+    API_KEY = '12345'
+    COST_HISTORY_REPORT = 'olap_reports/cost/history'
+    COST_CURRENT_REPORT = 'olap_reports/cost/current'
+    
+    @patch('krux_cloud_health.cloud_health.get_stats')
+    @patch('krux_cloud_health.cloud_health.get_logger')
+    def setUp(self, mock_logger, mock_stats):
+        self.mock_logger = mock_logger
+        self.mock_stats = mock_stats
+        self.cloud_health = CloudHealth(CloudHealthTest.API_KEY, self.mock_logger, self.mock_stats)
```

Comment:
This doesn't make sense. `self.mock_logger` is `get_logger()` function. You are passing a function as a logger to `CloudHealth`? While this would probably work because they are all MagicMock's, this doesn't make much sense.